src/stores/Store.ts

```diff
@@ -74,6 +78,184 @@ export default class Store {
 
 	@observable isSchemaApplied = false;
 
+	@observable parsedMessagesHistory: string[] = [];
```

Comment:
i suggest we store messages parsed and only stringify when it's needed.
it's type can be any object with sessionId, messageType and dictionary